src/main/java/jalau/at18/katas/multiples/deniso/SumMultiples.java

```diff
@@ -0,0 +1,18 @@
+package jalau.at18.katas.multiples.deniso;
+public class SumMultiples {
+
+    private static final int NUMBER_SIZE = 10;
+    private static final int NUMBER_ONE = 3;
+    private static final int NUMBER_TWO = 5;
```

Comment:
Constant names need to be improved, they should reflect how those values are being used, for example, MULTIPLIER_FIVE